.scripts/deploynaut-php-resque

```diff
@@ -25,6 +25,7 @@ PIDFILE=/var/run/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
 USERNAME=www-data
 LOGFILE=/var/log/$NAME
+#STATUSFILE=$ROOT/resque_status.sh
```

Comment:
And that shouldn't be commented out I think, because you are doing a `[ -f "$STATUSFILE" ]` check below anyway. Noone will remember to comment this out before installing.